fe/fe-core/src/main/java/org/apache/doris/analysis/InsertOverwriteTableSelect.java

```diff
@@ -0,0 +1,69 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.analysis;
+
+import org.apache.doris.common.ErrorCode;
+import org.apache.doris.common.ErrorReport;
+import org.apache.doris.common.UserException;
+
+import lombok.Getter;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class InsertOverwriteTableSelect extends DdlStmt {
+    @Getter
+    private final CreateTableLikeStmt createTableLikeStmt;
+
+    @Getter
+    private final InsertStmt insertStmt;
+    @Getter
+    private final AlterTableStmt alterTableStmt;
+    @Getter
+    private QueryStmt queryStmt;
+
+    public InsertOverwriteTableSelect(CreateTableLikeStmt createTableLikeStmt, QueryStmt queryStmt) {
```

Comment:
The constructor of `InsertOverwriteTableSelect` should same as `InsertStmt`.
And `CreateTableLikeStmt` and `QueryStmt` should be generated in analyze phase, not from cup file